hangupsbot/plugins/telesync/__init__.py

```diff
@@ -24,6 +24,9 @@
 logger = logging.getLogger(__name__)
 
 
+# prefix for profile sync registration, must be non-empty string
+reg_code_prefix = "VERIFY"
```

Comment:
telesync was creating random numbers that looked very much like hangouts and telegram user ids as part of the registration process and not clearing it out properly - adding a prefix ensures that these entries can be instantly recognised by the system